README.md

```diff
@@ -127,6 +128,13 @@ accomplish this as follows:
         ```http
         Sec-CH-UA-Model: "Pixel 2 XL"
         ```
+
+    5.  The `Sec-CH-UA-Mobile` header field represents whether the user agent should receive a specifically "mobile"
```

Comment:
Browsers distinguish between more than just Mobile or not Mobile -- there are also `VR` (and sometimes `Mobile VR` (some oculus and samsung devices) and `TV` `<devicecompat>` tokens (my phrase, I think Samsung calls it "[UX RECOMMEND](https://developer.samsung.com/internet/user-agent-string-format)") in user agent strings, depending on form factor. Rather than a new boolean for each of those, it's probably better to abstract over that, if this extension seems useful.

Reply:
Framing it in terms of UX recommendations might make sense. I like that semantic more than "This is mobile."

I wonder what granularity we could reasonably provide by default, though. This hint only makes sense if we're sending it by default, as developers can of course request actual viewport information directly. I worry that too many buckets will add enough entropy to make it unlikely we'd benefit users by shipping this without an opt-in.

Reply:
I was considering this too; making an enum or list of values as opposed to just a boolean. I liked that it was just one bit of information, but it may not be useful.

I do agree with @mikewest that if we make it an enum or even just a string, the balance of of entropy to usability will be rough. I also worry that feature creep might turn it into a mess of different values, way too specific values, and general compat mess.

There's also the issue that if someone decides to make a new value, no one else will use it and the default might be more or less useful than another more popular option (like if you're on a TV and the "mobile" version might be more useful the "desktop" version).

We could set up a preference order (like font-families in CSS), although that might make for more fingerprinting (?) it would make graceful failing of unknown values better.